docs/ui/components/Collapsible/index.tsx

```diff
@@ -1,9 +1,16 @@
 import { css } from '@emotion/react';
-import { shadows, theme } from '@expo/styleguide';
+import { LinkBase, shadows, theme } from '@expo/styleguide';
 import { borderRadius, spacing } from '@expo/styleguide-base';
 import { TriangleDownIcon } from '@expo/styleguide-icons';
+import { useRouter } from 'next/router';
```

Comment:
```suggestion
import { useRouter } from 'next/compat/router';
```
We need to use router `compat` export when using router hook, this should address few of the test CI complains.